src/test/kotlin/com/greenbird/utilihive/stubs/sftp/SftpServerTestContextTest.kt

```diff
@@ -284,6 +284,64 @@ class SftpServerTestContextTest {
             )
     }
 
+    // Absolute vs relative path tests
```

Comment:
In these test, is it not more relevant to test with getPath instead of directly on the server with downloadFile and verify expected behaviour of absolute and relative paths that way? That is the API we provide for the user, and that is the API we need to verify works as expected.

Reply:
Good point, changed it.